frontend/apps/filemanager/filemanagerfilesearcher.lua

```diff
@@ -126,10 +126,10 @@ function FileSearcher:close()
     end
 end
 
-function FileSearcher:onShowFileSearch()
+function FileSearcher:onShowFileSearch(str)
```

Comment:
For function parameters, best to give explicite names, here just "input" or "pattern" or "search_string", instead of "str".
("str" might be ok for local variables)